src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs

```diff
@@ -12,7 +12,7 @@ namespace Tizen.NUI.Binding
     /// Provides a mechanism by which application developers can propagate changes that are made to data in one object to another, by enabling validation, type coercion, and an event system.
     /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public abstract class BindableObject : INotifyPropertyChanged, IDynamicResourceHandler
+    public abstract class BindableObject : Disposable, INotifyPropertyChanged, IDynamicResourceHandler
```

Comment:
I am not sure this is a good idea. View inherits BindableObject but it already implements IDisposable in BaseHandle.
In my opinion, duplicate implements would cause additional build warnings.